### ISSUE TITLE
Undefined color in PDF compile

### ISSUE THREAD

**Issue by ClassRepo** [openmainframeproject/cobol-programming-course#165]
I am getting this response upon compile:

```
Error producing PDF.
! LaTeX Error: Undefined color `black!10'.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...

l.914 \begin{lstlisting}
```
I found the "offending" line in the markdown file:

```
// original line 8
    backgroundcolor=\color{black!10},
```

I modified it to this:

```
// modified line 8
    backgroundcolor=\color{black},
```

I have no idea if that's the same as what you wanted. It does compile, and seems to be ok.

I believe this is where the black background is used—didn't think to look until after I posted this--(I left it larger than I would like so you could expand it and see the details better):

[Image of compiled section](https://postimg.cc/cgxQC4n4)

Using MacTex-BasicTex, as MikTex breaks (or fails, not sure which) on Mac.

**Comment by MikeBauerCA** [comment]
If you are just trying to access the content, all releases of the course are published at https://github.com/openmainframeproject/cobol-programming-course/releases.

**Comment by mpettis** [comment]
@MikeBauerCA I looked at the link, and I was able to get the PDF for the first course (COBOL-Programming-Course-1-Getting-Started.pdf), but not the second or third.  Is there a place that all of the course PDFs are built?

**Comment by MikeBauerCA** [comment]
@mpettis the second and third courses are still in development. No official PDF distributions are available yet. When they are available, they will be included in the releases.

**Comment by MikeBauerCA** [comment]
@mpettis the second course PDF was included in the most recent release: [2.1.0](https://github.com/openmainframeproject/cobol-programming-course/releases/tag/2.1.0).
